### DISCUSSION TITLE
NodeNotFoundError

### DISCUSSION THREAD

**Discussion by EGrun** [noop-inc/documentation#141] [Bugs]
<img width="720" alt="image" src="https://github.com/noop-inc/documentation/assets/3859088/b36fbafc-d51c-4c9d-8085-6ec751d88226">

After clicking start new Deployment and selecting a previous Build to deploy

**Comment by 1800joe** [comment]
Interesting. Was this just after you opened or restarted the app?

**Comment by EGrun** [comment]
It was soon after a restart.

**Comment by 1800joe** [comment]
I can't say for sure, without more info. What _could have_ happened is the local state refreshed without refreshing the UI, essentially creating a mismatch between the entities that the console receives updates about and the entities that are visible on screen. 

I'll put on the board to see if we can validate one way or the other in the latest release.

Thanks!

**Comment by derekwolpert** [comment]
@1800joe Mildly interesting here is the NodeNotFoundError is for a nonexistent Cloud Organization - NOT the Workshop Organization ID. Right now I'm not aware of a circumstance where the Workshop Organization would be accidentally queried for as a Cloud Organization, but it appears there might be a lingering edge-case here.

I'll keep an eye out for this issue. Let me know if anyone comes across a reliable way to reproduce.
